tools/Customization/DevHome.Customization/Strings/en-us/Resources.resw

```diff
@@ -205,6 +229,10 @@
     <value>Move contents of {0} to a directory on Dev Drive such as {1} and set {2} to that chosen directory on Dev Drive</value>
     <comment>Optimizer dev drive description when environment variable is not set</comment>
   </data>
+  <data name="PackageCachesDescription" xml:space="preserve">
+    <value>A package cache is the global folder location used by application to store files for installed software. These source files are needed when you want to update, uninstall or repair the installed software. Visual Studio is one such application that stores a large portion of it's data in the Package Cache.</value>
```

Comment:
"It's" is incorrect here. Also, should it be "used by *an* application"? The wording sounds weird there.
```suggestion
    <value>A package cache is the global folder location used by application to store files for installed software. These source files are needed when you want to update, uninstall, or repair the installed software. Visual Studio is one such application that stores a large portion of its data in the Package Cache.</value>
```